app/controllers/adjustments_controller.rb

```diff
@@ -0,0 +1,37 @@
+class AdjustmentsController < ApplicationController
+  include Administrated
+  layout 'admin'
+  before_action :set_adjustment, only: [:update]
+  
+  def edit
+    @adjustment = Adjustment.find(params[:id])
+  end
+
+  # PATCH/PUT /adjustments/1
+  # PATCH/PUT /adjustments/1.json
+  def update
+    @adjustment = Adjustment.find(params[:id])
+    respond_to do |format|
+      puts "----------------reason--#{@adjustment.reason}"
```

Comment:
@getbez may i suggest to use rails logger class. Not that this is a problem but the logger class provides really nice way of spitting information to rails log.